cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocConf.java

```diff
@@ -243,6 +267,69 @@ SSHPublicKey getPublicKey() {
     return publicKey;
   }
 
+
+  @Nullable
+  Map<String, String> getClusterMetaData() {
+
+
+
+    if (Strings.isNullOrEmpty(clustermetadata)) {
+
+      return Collections.emptyMap();
+    }
+
+
+    return StreamSupport.stream(KV_PARSER.parse(clustermetadata).spliterator(), false)
+            .collect(Collectors.toMap(
+                    KeyValue::getKey,
+                    KeyValue::getValue,
+                    (o, n) -> n,
+                    LinkedHashMap::new));
+  }
+
+
+  @Nullable
+  List<String> getNetworkTags() {
+
+    if (Strings.isNullOrEmpty(networktags)) {
+      return Collections.emptyList();
+    }
+
+    return Arrays.stream(Objects.requireNonNull(networktags).split(","))
```

Comment:
`requireNonNull` call is not needed since it was checked right above.

Reply:
Done